src/olympia/scanners/tasks.py

```diff
@@ -221,29 +220,82 @@ def _run_yara_for_path(scanner_result, path, definition=None):
 
 @task
 @use_primary_db
-def run_yara_query_rule_on_version(query_rule_pk, version_pk):
+def mark_yara_query_rule_as_completed(query_rule_pk):
     """
-    Run a specific ScannerQueryRule on a Version.
+    Mark a ScannerQueryRule as completed
+    """
+    rule = ScannerQueryRule.objects.get(pk=query_rule_pk)
+    rule.update(state=COMPLETED)
```

Comment:
Should we handle non-existent rules?

Reply:
I think we don't have to: if it doesn't exist somehow (and I'm using `use_primary_db` to avoid replica delay) then the task chord will fail, it's likely `run_yara_query_rule` will fail and the state will never be updated.

In a follow-up branch I'm adding ability to abort and that should take care of most error cases.